src/pages/Cart/Cart.js

```diff
@@ -0,0 +1,169 @@
+import React, { useEffect, useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import './Cart.scss';
+import CartList from './CartList';
+import TotalAmount from './TotalAmount';
+
+const Cart = () => {
+  const [items, setItems] = useState([]);
+  const [inputCheck, setInputCheck] = useState(true);
+  const qtyZero = items.filter(item => item.qty === 0);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/Shinung/data.json')
+      .then(res => res.json())
+      .then(result => setItems(result));
+  }, []);
+
+  const onDelete = () => {
+    const checkedItem = items.filter(item => item.is_check === true);
```

Comment:
```suggestion
    const checkedItem = items.filter(item => item.is_check);
```
item.is_check자체가 이미 boolean 인 것 같네요!

Reply:
> item.is_check자체가 이미 boolean 인 것 같네요!

앗! 그렇네요! 안써도 되는 것을 썼네요 ㅋㅋㅋ 감사합니다! 바로 반영하였습니다!